server/test/support/ResourceCreator.java

```diff
@@ -37,7 +37,7 @@ public ApiKey createActiveApiKey(String name, String keyId, String keySecret) {
         new ApiKey()
             .setName(name)
             .setKeyId(keyId)
-            .setExpiration(Instant.now().plusSeconds(1))
+            .setExpiration(Instant.now().plusSeconds(100))
```

Comment:
time based values like this in test setup can lead to inconsistent results, eg flakey tests if the real time is being used etc etc.

I'd suggest setting this to a "really long time"/infinite in the default case and set it shorter/adjust the test clock/etc in the tests that care about expiration.

Also comment that it effectively never expires.

Reply:
changed it to `Instant.MAX` and added a docstring for the method

Reply:
turns out Instant.MAX is out of range for a postgres timestamp, switched to making a year in the future and used a constant.